src/wx/autoupdater/wxmsw/wxvbam.rc

```diff
@@ -1,58 +1,58 @@
-#pragma code_page(65001)

```

Comment:
We cannot move this file, because it is needed for all windows builds, not just with winsparkle.

Just update the include paths.

If we add other implementations for windows, we can always do the necessary logic with cpp macros.

Reply:
This is the reason for the ci failure btw.